### PR TITLE
bumping up to concat 2.1.0 since that is the working version these days

### PR DESCRIPTION
**Pull request by supernovae**
most people will have the fixed concat these days, just bump up the version to allow it to work.

### CODE REVIEW

**Comment by jlambert121** [comment]
Could you squish these down into a single commit?  Also I'm fine with the puppetserver 2.3.x bump, could you update the README to reflect that as well?

**Comment by jlambert121** [comment]
Also can you bump up the minimum concat version?

Closes #49

**Comment by supernovae** [comment]
yeah, i'll squish these down and update readme.  i originally was just pushing for the concat fix but realized the puppetserver was missing a parameter so happy to roll them all together.

**Comment by supernovae** [comment]
note to self, don't hack on git rebase before coffee :)

File: metadata.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     }
   ],
   "dependencies": [
-    { "name": "puppetlabs/concat", "version_requirement": ">= 1.1.0 <2.0.0" },
+    { "name": "puppetlabs/concat", "version_requirement": ">= 1.1.0 <=2.1.0" },
     { "name": "puppetlabs/stdlib", "version_requirement": ">=4.2.0 <5.0.0" },
```
Comment (jlambert121) [inline metadata.json]: Since PL is pretty good with semver I think this should actually be <=3.0.0 -- sorry I didn't catch it before.

**Comment by jlambert121** [comment]
So it's up to 9 commits now -- maybe easier to just grab a diff and apply to a new branch?  (also I missed the version comment above, sorry)

**Comment by supernovae** [comment]
yeah, new PR coming. sorry.

**Pull request closed without merging**